src/utils/__tests__/date-test.js

```diff
@@ -28,7 +28,7 @@ describe('longDate', () => {
 
 describe('daysInDate', () => {
   test('determines the days in a Date', () => {
-    expect(daysInDate(new Date(1970, 0, 1, 2, 0))).toBe(0);
+    expect(daysInDate(Date.UTC(1970, 0, 1, 0, 0))).toBe(0);
```

Comment:
Just replacing `expect(daysInDate(new Date(1970, 0, 1, 2, 0))).toBe(0);` with `expect(daysInDate(Date.UTC(1970, 0, 1, 2, 0))).toBe(0);` will also work.

Reply:
Yeah. I checked. Both work fine. We can keep any.